generators/client/templates/angular/src/main/webapp/index.html.ejs

```diff
@@ -30,6 +30,7 @@
     <link rel="shortcut icon" href="favicon.ico" />
     <link rel="manifest" href="manifest.webapp" />
     <link rel="stylesheet" href="content/css/loading.css">
+    <script src="env.js"></script>
```

Comment:
I think it will memoize these settings on the user browser. What's our recommendation to reflect update in the `env.js` values?

Reply:
@vishal423 as you might think, this env.js file is mainly is pretty static, (unless you are debugging...) so In the use cases I though about this wasn't much of a problem. in case you have a use case where this file changes a lot (not only for debugging purposes) you can (I guess) configure webpack to add a hash in the filename...